finality-aleph/src/validator_network/incoming.rs

```diff
@@ -65,7 +65,8 @@ pub async fn incoming<D: Data, S: Splittable>(
     result_for_parent: mpsc::UnboundedSender<(AuthorityId, oneshot::Sender<()>)>,
     data_for_user: mpsc::UnboundedSender<D>,
 ) {
+    let addr = stream.peer_address_info();
     if let Err(e) = manage_incoming(authority_pen, stream, result_for_parent, data_for_user).await {
-        info!(target: "validator-network", "Incoming connection failed: {}", e);
+        info!(target: "validator-network", "Incoming connection failed: {}. Peer address info: {}", e, addr);
```

Comment:
I would go with "Incoming connection from {} failed: {}", this will look weird when there are errors as addresses, but much more natural in the overwhelming majority of cases.

Reply:
fixed